pkg/backupcompaction/api.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2025 The Cockroach Authors.
+//
+// Use of this software is governed by the CockroachDB Software License
+// included in the /LICENSE file.
+
+package backupcompaction
+
+import (
+	"context"
+
+	"github.com/cockroachdb/cockroach/pkg/sql/sem/eval"
```

Comment:
It looks like you followed the same approach that pcr used to avoid the data loop for a builtin, but in imho, it feels like a bit of overkill to add a new directory to `pkg`. But also, i don't really know where it should live-- i don't see a great precedent for this. When David's back from vacation, we should ask him and maybe the sql team.

Reply:
Updated to add a package-global function in `builtins`!

Reply:
🚀